collectors/python.d.plugin/mysql/README.md

```diff
@@ -17,7 +17,7 @@ To create the `netdata` user, execute the following in the MySQL shell:
 
 ```sh
 create user 'netdata'@'localhost';
-grant usage on *.* to 'netdata'@'localhost';
+grant replication client on *.* to 'netdata'@'localhost';
```

Comment:
```suggestion
grant usage, replication client on *.* to 'netdata'@'localhost';
```

Reply:
I think we shouldn't remove `usage`

Reply:
It's always granted implicitly but we can keep it for clarity, of course.